UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -97,6 +100,7 @@ void ResetMatrixManager()
 		Instance.mainStationMatrix = null;
 		MovableMatrices.Clear();
 		ActiveMatrices.Clear();
+		ActiveMatricesList.Clear();
```

Comment:
ActiveMatricesList  yeah it is
reason why it is so you can do for i Loop that's better before performance

Reply:
There's only half a dozen or so entries in the dictionary right? so I don't think a list would do much. Seems like the difference of iterating dictionary values vs list is quite marginal?

Reply:
I dont think ActiveMatrices and ActiveMatricesList should exist at the same time, its the one or the other

Reply:
though remember that it is called like 50 times Frame roughly

Reply:
I think it's fine to have both